Documentation/ContentBlocks/Index.rst

```diff
@@ -147,12 +155,23 @@ to format and sanitize the output.
 Line 5: We use the `Link.typolink ViewHelper <f:link.typolink> <https://docs.typo3.org/permalink/t3viewhelper:typo3-fluid-link-typolink>`_
 to render a link to the target that was defined in our custom field `button_link`.
 
+..  _content-blocks-carousel:
+
+The carousel Content Block
+==========================
+
+The carousel Content block is provided in the generated site package as an
+example on how to create a content block that contains on repeated items, here a
+carousel with multiple slides.
+
+It is described in chapter `Carousel Content Block example <https://docs.typo3.org/permalink/t3sitepackage:carousel>`_.
+
 ..  _content-blocks-next:
 
 Next steps
 ==========
 
-*   The generated site package contains a second, more elaborate example in
-    directory :path:`packages/my_site_package/ContentBlocks/ContentElements/carousel`.
+*   The generated site package contains a second, more elaborate example, the
+    :ref:`Carousel <carousel>`.
```

Comment:
The link is the exact same link than from line 167 above, but with different link text.

Reply:
I removed the sentence, not nessesary any more